src/Whoops/Resources/css/whoops.base.css

```diff
@@ -461,6 +463,38 @@ kbd {
 }
 
 pre.sf-dump {
-    padding: 0px !important;
-    margin: 0px !important;
-}
\ No newline at end of file
+  padding: 0px !important;
+  margin: 0px !important;
+}
+
+.search-for-help {
+  width: 85%;
+  padding: 0;
+  margin: 10px 0;
+  list-style-type: none;
+  display: inline-block;
+}
+  .search-for-help li {
+    display: inline-block;
+    margin-right: 10px;
+  }
+  .search-for-help li:last-child {
+    margin-right: 0;
+  }
+    .search-for-help li a {
+
+    }
+      .search-for-help li a i {
+        width: 16px;
+        height: 16px;
+        overflow: hidden;
+        display: block;
+      }
+      /* http://creativecommons.org/licenses/by/3.0 */
```

Comment:
This is a good start, but it's still not good enough. First of all, nobody has any way to check whether this is right. Who made the icon, where does it say that this license apply to it? Secondly, the license in question, like the vast majority of them, requires attribution. So at the very least the name of the author, collection name, URL of the website must be here as well. Here is how it should generally look like:

```php
/**
 * Cog icon by Hanan Awada, from the Noun Project
 * https://thenounproject.com/term/cog/309446/
 * Licensed under Creative Commons BY 3.0 US
 * http://creativecommons.org/licenses/by/3.0/us/
 */
```